Resources/Prototypes/Radium/Roles/Antags/thiefvoxes.yml

```
@@ -0,0 +1,24 @@
+    back: ClothingBackpackDuffelMilitary
+    ears: ClothingHeadsetAltVox
+    gloves: ClothingHandsGlovesCombat
+    outerClothing: ClothingOuterVoxThief
+    shoes: ClothingShoesBootsCombatFilled
+    id: PiratePDA
+    pocket2: AgentIDCard
+  storage:
+    back:
+    - ChameleonProjector
+    - Truncheon
+    - HypoDartBox
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)